dockerfile_parse/parser.py

```diff
@@ -442,7 +474,12 @@ def _instruction_getter(self, name, env_replace):
                         envs[key] = value
 
         logger.debug("instructions: %r", instructions)
-        return Labels(instructions, self) if name == 'LABEL' else Envs(instructions, self)
+        if name == 'LABEL':
+            return Labels(instructions, self)
+        elif name == 'ENV':
+            return Envs(instructions, self)
+        else:
```

Comment:
Would be possible to make it explicit `elif name == 'ARG':`  and keep compatibility with `else` branch returning `Envs`?